phpcs.xml

```diff
@@ -1,4 +1,10 @@
 <?xml version="1.0"?>
-<ruleset>
+<ruleset name="PHPCS"
+         xmlns="http://pmd.sf.net/ruleset/1.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://pmd.sf.net/ruleset/1.0.0 http://pmd.sf.net/ruleset_xml_schema.xsd"
+         xsi:noNamespaceSchemaLocation="http://pmd.sf.net/ruleset_xml_schema.xsd">
+    <description>PHPCS</description>
+    <!--<exclude-pattern>path/to/exclude/*</exclude-pattern>-->
     <rule ref="./vendor/mediact/coding-standard/src/MediaCT"/>
```

Comment:
Is this going to be updated at a later stage?

Reply:
yes